jf_agent/jf_jira/jira_download.py

```diff
@@ -384,10 +384,14 @@ def _download_some(thread_num, start_at, end_at):
                             )
                             continue
                         else:
-                            agent_logging.log_and_print_error_or_warning(
-                                logger, logging.ERROR, error_code=3022,
+                            # copied logic from jellyfish direct connect
+                            # don't bail, just skip
+                            # khardy 2023-03-16
+                            agent_logging.log_and_print_error_or_warning(logger, logging.WARNING, msg_args=
+                                f"Caught {type(e)} from search_issues(), batch size is already 0, giving up on "
+                                "fetching this issue's metadata", error_code=3092
                             )
-                            raise
+                            start_at += 1
```

Comment:
Also I think at this point we'll have reduced batch size to 0, so on the next iteration of the loop the batch size will still be 0; so I'm not sure that subsequent iterations are going to actually fetch the rest of the issues. Resetting the batch size back up to its starting point may be necessary. 🤔